### DISCUSSION TITLE
Create a standard way to define extension image previews

### DISCUSSION THREAD

**Discussion by pmareke** [zed-industries/zed#23911] [Feature Requests]
I think it will be really nice to have the preview of the extension inside zed or even in the future in a web app marketplace.

I was trying to recollect all the images but each extension repository has a different approach to host the images (some of them don't have a photo).

Taking a look to the current extensions all of them should have an [extension.json](https://github.com/e-simpson/new-darcula-z/blob/9fa13b1726d8ee9c7fffbc598d76a49b17a1c460/extension.json) file which I think is the perfect place to add a new field called `image` or something similar.

What do you think about it? I don't want to make noise in the amazing repository, so please let me know if this is not the place and I'll remove the issue.

**If you see some value on it I can do the pull requests in all the extensions.**

Thanks for all team.

**Comment by mgsloan** [comment]
I really like this idea, especially for themes!  We're moving feature requests to discussions.
